ADQL.tex

```diff
@@ -348,17 +348,15 @@ \subsubsection{Literals}
         <quote> [ <character_representation>... ] <quote>
 \end{verbatim}
 
-Numeric literals may be are expressed as an exact decimal value, e.g. \verb:12: or \verb:12.3:,
-a floating point number with an exponent, e.g. \verb:12.3E4:, or a unsigned hexadecimal
-value, e.g. \verb:0x2F:.
+Numeric literals are expressed as an exact decimal value, e.g. \verb:12: or
+\verb:12.3:, a floating point number with an exponent, e.g. \verb:12.3E4:.
```

Comment:
Should have an or in here "or a floating..."

Reply:
Fixed